@here/harp-mapview/lib/FrustumIntersection.ts

```diff
@@ -186,9 +194,6 @@ export class FrustumIntersection {
 
             // Stop subdivision if hightest visible level is reached
             const tileKey = tileEntry.tileKey;
-            if (tileKey.level > maxTileLevel) {
```

Comment:
You removed this, which is good, you now need to add the check to not add the item if it is past the maxTileLevel, i.e. before `workList.push(subTileEntry);`... or do I miss something?

Reply:
I don't think that's necessary. The `maxTileLevel` is determined by the highest `displayZoomLevel` of a data source. The method `DataSource.shouldSubdivide` checks if a tile key has reached the `displayZoomLevel` for this data source or not. So a tile key that is already at the right zoom level is not further divided.

Reply:
Good point! I saw the `maxTileLevel` still in the param list as well, but it seems you have fixed that too.